war/.gitignore

```diff
@@ -1,20 +0,0 @@
-work
```

Comment:
Unneeded now that the corresponding entries are in the root directory's `.gitignore` file.